modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessResourceImpl.java

```diff
@@ -88,6 +95,53 @@ public Process getProcess(Long processId) throws Exception {
 		);
 	}
 
+	@Override
+	public Date getProcessLastCheckedDate(Long processId) throws Exception {
+		SearchSearchRequest searchSearchRequest = new SearchSearchRequest();
```

Comment:
@MarinhoFeliphe rename to `getProcessLastSLACheckedDate` or something else that mention that is related to SLA checking. pls